nixos/modules/installer/tools/nixos-version.sh

```diff
@@ -1,23 +1,36 @@
 #! @runtimeShell@
 
+if [ $# -eq 0 ]; then
```

Comment:
This is a potentially breaking change, since by default every other parameter would output the default. Nevertheless, it is much better to be explicit and throw errors, if the user mistyped.